skyve-ee/src/skyve/modules/admin/ControlPanel/views/_generateTestData.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<view title="Control Panel" name="_generateTestData" xmlns="http://www.skyve.org/xml/view" xsi:schemaLocation="http://www.skyve.org/xml/view ../../../../schemas/view.xsd"
+	xmlns:c="http://www.skyve.org/xml/common" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+	<vbox border="true" borderTitle="Generate Test Data">
+		<form>
+			<column responsiveWidth="3" />
+			<column pixelWidth="50" />
+			<column />
+			<column responsiveWidth="1" />
+			<column responsiveWidth="3" />
+			<row>
+				<item colspan="3">
+					<combo binding="testModuleName">
+						<onChangedHandlers>
+							<rerender clientValidation="false" />
+						</onChangedHandlers>
+					</combo>
+				</item>
+			</row>
+			<!-- <row>
```

Comment:
Can you remove these commented lines, these are from before we added the listMembership so aren't required anymore